deployment/roles/common/tasks/repo.yml

```diff
@@ -1,12 +1,12 @@
 ---
-- name: Get bridge repo
-  git:
-    repo: "{{ bridge_repo }}"
+- name: Create repo directory
+  file: 
+    path: "{{ bridge_path }}"
+    state: directory
+- name: Synchronize the repository with current files
```

Comment:
It's better to split tasks with a newline.
Also I would agree with @akolotov on using `ls-tree` command

Reply:
@ArseniiPetrovich Done, please review again.